.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -42,7 +42,7 @@ Otherwise your PR may be subject to ARM review requirements. Complete the follow
 ### Breaking Change Review Checklist
 If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
 
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](http://aka.ms/aprwiki).
```

Comment:
I added a commit removing this broken URL:

https://msazure.visualstudio.com/One/_git/Azure-Documents-Common/commit/7e5a5d074141ab3ffa396a9f9bd82cefdf782fa3?refName=refs%2Fheads%2FCPE-SLA

Not sure how/when it will be deployed. Hopefully soon.

Reply:
OK, [the CI kicked in](https://eng.ms/content/385c0459-4e49-493e-b827-9915db965ef2/admin/29d02a1a-6c6b-47fe-8fa1-0c5b46ef69c7/OrchestrateScheduleGeneration) and the changes are reflected. So we can update these eng.ms docs without any reviews.